crates/chain/src/lib.rs

```diff
@@ -97,5 +100,58 @@ pub mod collections {
     pub use hashbrown::hash_map;
 }
 
+/// Number of seconds that have elapsed since 00:00:00 UTC on 1 January 1970, the Unix epoch.
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Debug, Clone, PartialEq, Eq, Copy, PartialOrd, Ord, core::hash::Hash)]
+pub struct UnixSeconds(u64);
+
+impl UnixSeconds {
+    pub fn from_consensus(n: u64) -> UnixSeconds {
+        UnixSeconds(n)
+    }
+}
+impl Default for UnixSeconds {
+    fn default() -> Self {
+        UnixSeconds(0)
+    }
+}
+
+#[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
+#[derive(Debug, Clone, PartialEq, Eq, Copy, PartialOrd, Ord, core::hash::Hash)]
+pub struct BlockHeight(Height);
+
+impl Default for BlockHeight {
+    fn default() -> Self {
+        return BlockHeight(
+            Height::from_consensus(0).expect("Failed to create a default BlockHeight"),
+        );
+    }
+}
+
+impl Deref for BlockHeight {
```

Comment:
[This is an anti-pattern](https://rust-unofficial.github.io/patterns/anti_patterns/deref.html)